test_chunking.py

```diff
@@ -0,0 +1,81 @@
+from minions.clients.ollama import OllamaClient
```

Comment:
would be great if you could have this script moved to a new `tests/` folder, and have the `pytest` syntax to assert things strictly